python/export_util.py

```diff
@@ -0,0 +1,56 @@
+from typing import List
+import mgp
+import json as js
+
+
+@mgp.read_proc
+def json(ctx: mgp.ProcCtx, path: str) -> mgp.Record():
+    """
+    Procedure to export the whole database as JSON to a local file.
+
+    Parameters
+    ----------
+    path : str
+        Path to the file where JSON will be saved.
+    """
+    nodes = list()
+    relationships = list()
+    graph = list()
+
+    for vertex in ctx.graph.vertices:
+        labels = [label.name for label in vertex.labels]
+        properties = dict()
```

Comment:
`properties = {key: vertex.properties.get(key) for key in vertex.properties.keys() }`

Just the proposal

Reply:
I like the proposal so I have changed it :)